#! /bin/sh /usr/share/dpatch/dpatch-run
## 11_format_string_error.dpatch by Stefan Fritsch <sf@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: backport from 1.5.2

@DPATCH@
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' mp3gain-1.5.1~/rg_error.c mp3gain-1.5.1/rg_error.c
--- mp3gain-1.5.1~/rg_error.c	2008-09-15 14:08:28.000000000 +0200
+++ mp3gain-1.5.1/rg_error.c	2010-08-28 20:18:07.812558320 +0200
@@ -40,7 +40,7 @@
 {
 #ifndef asWIN32DLL
     gSuccess = 0;
-	fprintf(stdout, localerrstr);
+	fprintf(stdout, "%s", localerrstr);
 #else
 	//send message to DLL's parent
     mp3gainerr = localerrnum;
